protocol/src/main/resources/definitions.json

```diff
@@ -6,6 +6,10 @@
     "did": {
       "type": "string",
       "pattern": "^did:([a-z0-9]+):((?:(?:[a-zA-Z0-9._-]|(?:%[0-9a-fA-F]{2}))*:)*((?:[a-zA-Z0-9._-]|(?:%[0-9a-fA-F]{2}))+))((;[a-zA-Z0-9_.:%-]+=[a-zA-Z0-9_.:%-]*)*)(\/[^#?]*)?([?][^#]*)?(#.*)?$"
+    },
+    "decimalString": {
+      "type": "string",
+      "pattern": "^([0-9]+[.][0-9]+)$"
```

Comment:
Will this update the schema on `tbdex` repo as well?

Reply:
Yes it will, as part of this PR in `tbdex` https://github.com/TBD54566975/tbdex/pull/206/files#diff-1c7ad0f786a4285db57ee19390ddaf55366f700c3d9ff382a30a984dccca35dfR12